commands/info.md

```diff
@@ -182,15 +193,15 @@ Here is the meaning of all fields in the **stats** section:
      server
 *   `total_commands_processed`: Total number of commands processed by the server
 *   `instantaneous_ops_per_sec`: Number of commands processed per second
-*   `total_net_input_bytes`: TBD
-*   `total_net_output_bytes`: TBD
-*   `instantaneous_input_kbps`: TBD
-*   `instantaneous_output_kbps`: TBD
+*   `total_net_input_bytes`: The total number of bytes read from the network
+*   `total_net_output_bytes`: The total number of bytes written to the network
+*   `instantaneous_input_kbps`: The network's read rate per second in KB/sec
+*   `instantaneous_output_kbps`: The network's write rate per second in KB/sec
 *   `rejected_connections`: Number of connections rejected because of
      `maxclients` limit
-*   `sync_full`: TBD
-*   `sync_partial_ok`: TBD
-*   `sync_partial_err`: TBD
+*   `sync_full`: The number of full resyncs with slaves
+*   `sync_partial_ok`: The number of accpepted partial resync requests
```

Comment:
Type - "accpepted" should be "accepted"

Reply:
True - thanks!

Reply:
BTW, @boughtonp, it would be better if you can make future reviews at the origin branch so I can commit them directly to the PR.

Reply:
Ok, I'll keep that in mind for future.